src/core/src/pass/sdpa_to_vlsdpa.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/pass/sdpa_to_vlsdpa.hpp"
+
+#include <array>
+#include <string_view>
+#include <utility>
+
+#include "openvino/cc/pass/itt.hpp"
+#include "openvino/op/scaled_dot_product_attention.hpp"
+#include "openvino/pass/manager.hpp"
+#include "ov_ops/vl_sdpa.hpp"
+#include "transformations/utils/utils.hpp"
+
+using namespace ov::op;
+using namespace ov::pass;
+
+SDPAToVLSDPA::SDPAToVLSDPA() = default;
+
+namespace {
+
+std::shared_ptr<v0::Parameter> addName(std::shared_ptr<v0::Parameter> node, const char* name) {
```

Comment:
```suggestion
std::shared_ptr<v0::Parameter> add_name(std::shared_ptr<v0::Parameter> node, const char* name) {
```
The function can be removed there is not much use of it and there.
as in current context the input parameter is has no names.

use `ov::util::set_name` from `node_util.hpp`

If kept don't pass shared pointer as the counter is updated twice on input and output pass reference instead.